messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_12_0.xsd

```diff
@@ -303,6 +303,23 @@
                 </xs:complexType>
             </xs:element>
 
+            <xs:element maxOccurs="1" minOccurs="0" name="critical-analyzer">
+                <xs:complexType>
+                    <xs:attribute name="critical-analyzer-enabled" type="xs:boolean" />
```

Comment:
We should probably set default values for this and the other entries

Reply:
ok